metrics/exporter_test.go

```diff
@@ -95,6 +95,9 @@ func getResourceLabelValue(key string, tags []tag.Tag) string {
 
 func TestMain(m *testing.M) {
 	resetCurPromSrv()
+	// Set gcpMetadataFunc and newStackdriverExporterFunc for testing
```

Comment:
Since these are really only for Stackdriver exporter, could you move this to stackdriver_exporter_test.go?

Reply:
We can't have multiple `TestMain` func.

Reply:
oh, darn :(